docs/nb/bgs_main_map.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    plt.ylabel('Frequency')
</pre><p>Add BGS Faint and MWS_ANY use DESI_TAGET and desi_mask(MWS_ANY).  For maps and histograms.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahbigwood/S4Mock/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>